src/gtfs_flex_validation.py

```diff
@@ -38,16 +45,17 @@ def is_gtfs_flex_valid(self) -> tuple[Union[bool, Any], Union[str, Any]]:
         dest = DOWNLOAD_FILE_PATH
         is_valid = False
         validation_message = ''
-        if self.file_relative_path.endswith('.json') or self.file_relative_path.endswith('.zip'):
+        root, ext = os.path.splitext(self.file_relative_path)
+        if ext:
```

Comment:
why this ext check? and i dont understand the else part of this check. you should just check if the file ends with .zip and if not reject it.